app/api/workday/route.ts

```diff
@@ -0,0 +1,12 @@
+import { getWorkdayData } from "@/app/about/queries"
+
+interface WorkdayRequest extends Request {}
```

Comment:
WorkdayRequest is extending Request... but there isn't any modifications. This could be deleted, and then you could use `Request` a few lines below